internal/pkg/cli/interfaces.go

```diff
@@ -21,6 +20,7 @@ import (
 	"github.com/aws/copilot-cli/internal/pkg/aws/codepipeline"
 	awsecs "github.com/aws/copilot-cli/internal/pkg/aws/ecs"
 	"github.com/aws/copilot-cli/internal/pkg/aws/s3"
+	clideploy "github.com/aws/copilot-cli/internal/pkg/cli/deploy"
 	"github.com/aws/copilot-cli/internal/pkg/config"
 	"github.com/aws/copilot-cli/internal/pkg/deploy"
```

Comment:
man we have so many deploys 😂 not in this refactor obviously, but I think `pkg/deploy` is just noise. We can delete it and move almost all the logic to the existing packages like `manifest`, `cloudformation/stack` or `template`

Reply:
I'll add it to ongoing issues.